istio.io/configmap-nginx.yaml

```diff
@@ -149,6 +149,47 @@ data:
           return 301 https://istio.io$request_uri;
       }
 
+      # gcsweb.istio.io
```

Comment:
Looks like there is a some extra copy/paste errors here. I see two gcsweb.istio.io server blocks.

Reply:
I don't see it.

Reply:
Nevermind. I forget we defined separate http and https server blocks